changelog/2248.fixed.md

```diff
@@ -0,0 +1,2 @@
+Fixed solaris build globally, also removed a subset of `InterfaceFlags` values
+which are of i64 type.
```

Comment:
"which are of i64 type" still doesn't make sense to a typical reader.  Why were they removed?  Did they not work at runtime, or did they generate build errors?
Also, the ", also" creates a run-on sentence.  You should use a proper conjunction or split it into two sentences.

Reply:
That changelog edit still doesn't answer the question.  Did the interfaceflags fail at runtime, or generate build errors?  Also, why did you elect to remove some of them, rather than just change the field width to 64 bits on Solarish?